coalib/output/ConsoleInteraction.py

```diff
@@ -87,9 +87,13 @@ def highlight_text(no_color, text, lexer=TextLexer(), style=None):
 DIFF_EXCERPT_MAX_SIZE = 4
 
 
-def format_lines(lines, line_nr=''):
-    return '\n'.join('|{:>4}| {}'.format(line_nr, line)
-                     for line in lines.rstrip('\n').split('\n'))
+def format_lines(lines, symbol='', line_nr=''):
+    if symbol == '[':
+        return '\n'.join('{}{:>4}{} {}'.format(symbol, ']', line_nr, line)
+                         for line in lines.rstrip('\n').split('\n'))
+    else:
```

Comment:
theres no need for an else when you do:

if a:
    return
return

if the if condition is true, the function will return. if its not, it will execute the second return